### PR TITLE
Update OpenGL's backend version from 3.3 to 4.3

### PR DESCRIPTION
**Pull request by FernandoS27**

### CODE REVIEW

**Comment by FernandoS27** [comment]
@ReinUsesLisp done.

File: src/video_core/renderer_opengl/gl_rasterizer.h [hunk 1]
```diff
@@ -177,8 +177,8 @@ class RasterizerOpenGL : public VideoCore::RasterizerInterface {
 
     bool has_ARB_direct_state_access = false;
     bool has_ARB_multi_bind = false;
-    bool has_ARB_separate_shader_objects = false;
-    bool has_ARB_vertex_attrib_binding = false;
+    bool has_ARB_separate_shader_objects = true;
```
Comment (ogniK5377) [inline src/video_core/renderer_opengl/gl_rasterizer.h]: These two can be removed now since they don't seem to be used anywhere anymore?

Comment (FernandoS27) [reply src/video_core/renderer_opengl/gl_rasterizer.h]: yeah we could.

**Comment by FernandoS27** [comment]
@ogniK5377 changes made.

**Comment by bunnei** [comment]
Needs a rebase

**Comment by FernandoS27** [comment]
Updated.

**Pull request merged**